core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/LibraryDilution.java

```diff
@@ -95,6 +89,7 @@
   private Double concentration;
 
   private String concentrationUnits;
+  private String volumeUnits;
```

Comment:
Now that there's an enum, use enum types (and on other types). This will require updating the values in the DB, but then helps to prevent bad data

Reply:
Should the DB update go in a migration, or as a separate SQL query?

Reply:
migration